.github/workflows/linux.yml

```diff
@@ -37,6 +37,7 @@ jobs:
           - "3.9"
           - "3.10"
           - "3.11"
+          # - "3.12"
```

Comment:
It looks like Linux CI environment on GitHub only has Python 3.12.0.rc3 which causes conflicts with other packages. We will enable 3.12 once a proper 3.12.0 release is available on GitHub CI. #832 opened to track the issue

Reply:
I left some comments in the issue.  I think we might want to consider switching to conda-forge as the priority everywhere again to let this work.

Reply:
I want to see the error message so I pushed a small change to re-enable this for now.

Reply:
FWIW, we can also do this locally like so:

```sh
make CONDA_ENV_NAME=mlos-3.12 conda-env
```

Reply:
Indeed it was a conda-forge channel priority issue.

Reply:
Switching all of the conda env files over to that for consistency.